Jenkinsfile

```diff
@@ -34,8 +34,12 @@ pipeline {
 				script{
 					withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
 						def slice_test_snapshot_dump = "${env.SLICE_TEST}_${currentRelease}_snapshot.dump"
+						def slice_previous_snapshot_dump = "${env.SLICE_TEST}_${previousRelease}_snapshot.dump.gz"
+						// Retrieve snapshot DB from S3.
+						sh "aws s3 --no-progress cp ${env.S3_RELEASE_DIRECTORY_URL}/${previousRelease}/update_stable_ids/$slice_previous_snapshot_dump ."
 						sh "mysql -u$user -p$pass -e \'drop database if exists ${env.SLICE_PREVIOUS}; create database ${env.SLICE_PREVIOUS}\'"
-						sh "zcat  archive/${previousRelease}/${env.SLICE_TEST}_${previousRelease}_snapshot.dump.gz 2>&1 | mysql -u$user -p$pass ${env.SLICE_PREVIOUS}"
+						sh "zcat  $slice_previous_snapshot_dump 2>&1 | mysql -u$user -p$pass ${env.SLICE_PREVIOUS}"
```

Comment:
Just curious why STDERR is redirected to STDOUT here for the zcat command?

Reply:
So apparently, it actually isn't needed. I remember exploring this back when I was setting up jenkins, and this was the only way to make the piped zcat-mysql command work. Just now though when I removed it, it ran perfectly fine. Either an update to Jenkins fixed whatever issue I had been having with this situation, or there is a specific context where this is needed elsewhere in the release pipeline. Anyways, I've removed the redirection from this Jenkinsfile and I'll be testing this with the other steps as well.